reark/src/main/java/io/reark/reark/data/store/ContentProviderStore.java

```diff
@@ -107,17 +108,21 @@ protected void insertOrUpdate(T item, Uri uri) {
         return list;
     }
 
-    @Nullable
-    protected T queryOne(Uri uri) {
-        final List<T> queryResults = queryList(uri);
+    @NonNull
+    protected Observable<T> queryOne(Uri uri) {
+        return Observable.just(uri)
```

Comment:
This was an adaptation of the existing code and felt kinda hackish, so sounds good :+1: